proto/types/types.proto

```diff
@@ -169,3 +170,16 @@ message SignedHeader {
   Header header = 1;
   Commit commit = 2;
 }
+
+message ValidatorSet {
+  repeated Validator validators         = 1;
```

Comment:
Can these be `null`? i.e. should we allow nullable?

Reply:
I'd vote for not allowing nulls here

Reply:
this is keeping in line with the tendermint type. Will make the change to not allow nulls here.